pkg/systemd/logs.scss

```diff
@@ -72,6 +72,11 @@
     padding-block-start: var(--pf-global--spacer--md);
   }
 
+  // RTL: Pass through computer-generated text as-is
+  .pf-c-description-list__text {
+    @extend .ct-rtl-passthrough;
+  }
```

Comment:
I find this too magic and detached from the code. This isn't even a cockpit specific CSS class, but changes every table. If that was just for a demo, that's fine of course, and it sounds like that's what you meant in the description.

I.e. for the real PR, let's rather use `<bdi>` or the class in JSX, at the points where we know it's computer-generated content.